src/main/java/org/apache/maven/shared/verifier/Verifier.java

```diff
@@ -1566,38 +1567,77 @@ public void reset()
         }
     }
 
+    /**
+     * @deprecated will be removed without replacement,
+     * for arguments adding please use {@link #addCliArgument(String)}, {@link #addCliArguments(String...)}
+     */
+    @Deprecated
     public List<String> getCliOptions()
     {
-        return cliOptions;
+        return cliArguments;
     }
 
+    /**
+     * @deprecated will be removed
+     */
+    @Deprecated
     public void setCliOptions( List<String> cliOptions )
     {
-        this.cliOptions = cliOptions;
+        this.cliArguments = cliOptions;
     }
 
     /**
      * Add a command line argument, each argument must be set separately one by one.
      * <p>
      * <code>${basedir}</code> in argument will be replaced by value of {@link #getBasedir()} during execution.
      * @param option an argument to add
+     * @deprecated please use {@link #addCliArgument(String)}
      */
+    @Deprecated
     public void addCliOption( String option )
     {
-        cliOptions.add( option );
+        addCliArgument( option );
+    }
+
+    /**
+     * Add a command line argument, each argument must be set separately one by one.
+     * <p>
+     * <code>${basedir}</code> in argument will be replaced by value of {@link #getBasedir()} during execution.
+     *
+     * @param cliArgument an argument to add
+     */
+    public void addCliArgument( String cliArgument )
+    {
+        cliArguments.add( cliArgument );
     }
 
     /**
      * Add a command line arguments, each argument must be set separately one by one.
      * <p>
      * <code>${basedir}</code> in argument will be replaced by value of {@link #getBasedir()} during execution.
+     *
      * @param options an arguments list to add
+     * @deprecated
      */
+    @Deprecated
```

Comment:
This method has not been released, I think we can just get rid of it instead of deprecating it.

Reply:
Thanks - you are right - I will remove it

Reply:
Done in: https://github.com/apache/maven-verifier/commit/76b0ffe3228dbccdccbc0bdccb618059e6bee4b7